File: src/C#/CreationalPatterns/AbstractFactory/Furniture/Products/ScandinavianCabinet.cs
using Furniture.Interfaces;
using System;
using System.Collections.Generic;
using System.Text;

namespace Furniture.Products
{
    public class ScandinavianCabinet : IFurniture
    {
        public void ShowFurnitureStyle()
        {
            Console.WriteLine("I am a Scandinavian cabinet");
        }
    }
}

File: src/PHP/CreationalPatterns/AbstractFactory/Furniture/Products/ScandinavianCabinet.php
<?php

namespace CreationalPatterns\AbstractFactory\Furniture\Products;

use CreationalPatterns\AbstractFactory\Furniture\Interfaces\IFurniture;

class ScandinavianCabinet implements IFurniture {
    public function showFurnitureStyle(): void
    {
        echo "I am a Scandinavian cabinet\n";
    }
}